packages/grid-pro/src/vaadin-grid-pro-edit-column.js

```diff
@@ -286,6 +290,23 @@ class GridProEditColumn extends GridColumn {
     this._focusEditor(editor);
   }
 
+  _onMouseDown(editor) {
+    return (e) => {
+      this.__mouseDownDebouncer = Debouncer.debounce(this.__mouseDownDebouncer, animationFrame, () => {
```

Comment:
Why do we need a debouncer here? I think we could just call `event.preventDefault()` immediately to prevent `focusout` event from being fired.

Reply:
You are right. The debouncer shouldn't be needed. Removing it and adding `event.preventDefault()` seems to be enough. I still need to keep the same condition to avoid preventing default is the click happens inside the input element.